main.py

```diff
@@ -0,0 +1,41 @@
+import streamlit as st
+from transformers import ViltProcessor, ViltForQuestionAnswering
+from PIL import Image
+
+processor = ViltProcessor.from_pretrained("dandelin/vilt-b32-finetuned-vqa")
+model = ViltForQuestionAnswering.from_pretrained("dandelin/vilt-b32-finetuned-vqa")
+
+
+def forward_pass():
```

Comment:
Kind of unobvious behavior: you function process an image and text, but not accepting them in arguments.